docker-compose.yml

```diff
@@ -1,4 +1,5 @@
-version: "2"
+version: "3"
```

Comment:
Did we need to update the docker-compose version here for something you use in the rest of the change?

Reply:
We should be fine to use the latest and greatest here, as long as it works. The docker file is only used for development, just to mention that again.